simulation-ros/src/turtlebot2i/turtlebot2i_description/v-rep_model/README.md

```diff
@@ -0,0 +1,40 @@
+# Turtlebot2i V-rep model
+
+This folder contains the scene files e.g. **turtlebot2i.ttt** and Lua files for each component e.g. **turtlebot2i_lidar.lua**.
+
+The scene files are:
+
+1. mapir_lab_turtlebot2i.ttt
+2. turtlebot2i.ttt
+3. turtlebot2i_multirobot.ttt
+4. warehouse_turtlebot2i.ttt
+5. warehouse_turtlebot2i_v2.ttt
+6. warehouse_turtlebot2i_v3.ttt
+
+**Comment from Andrew: **
+
```

Comment:
Here we have some comments from @berezovskyi . @albertohata can you take a look?

Reply:
Yes. The usual wow in vrep is to have all scripts inside the .ttt. The issue with this is that those are binaries, and any modificatin to the scripts would require us to commit/push the whole file. For this reason, we chose to externalize most of the scripts, into those lua files.